tripal_chado/src/Plugin/Field/FieldType/ChadoAnalysisTypeDefault.php

```diff
@@ -147,60 +147,70 @@ public static function tripalTypes($field_definition) {
     $properties[] = new ChadoIntStoragePropertyType($entity_type_id, self::$id, 'record_id', $record_id_term, [
       'action' => 'store_id',
       'drupal_store' => TRUE,
-      'chado_table' => $base_table,
-      'chado_column' => $base_pkey_col,
+      'path' => $base_table . '.' . $base_pkey_col,
+      //'chado_table' => $base_table,
+      //'chado_column' => $base_pkey_col,
     ]);
 
     // Base table links directly
     if ($base_table == $linker_table) {
       $properties[] = new ChadoIntStoragePropertyType($entity_type_id, self::$id, self::$object_id, $linker_fkey_term, [
         'action' => 'store',
         'drupal_store' => TRUE,
-        'chado_table' => $base_table,
-        'chado_column' => $linker_fkey_col,
+        'path' => $base_table . '.' . $linker_fkey_col,
+        //'chado_table' => $base_table,
+        //'chado_column' => $linker_fkey_col,
         'delete_if_empty' => TRUE,
         'empty_value' => 0,
       ]);
     }
     // An intermediate linker table is used
     else {
       // Define the linker table that links the base table to the object table.
+      // (e.g., project_analysis.project_analysis_id)
       $properties[] = new ChadoIntStoragePropertyType($entity_type_id, self::$id, 'linker_id', $record_id_term, [
         'action' => 'store_pkey',
         'drupal_store' => TRUE,
-        'chado_table' => $linker_table,
-        'chado_column' => $linker_pkey_col,
+        'path' => $linker_table . '.' . $linker_pkey_col,
```

Comment:
Both the change and the original should work but I think ideally we specify the full join here to ensure that these properties are evaluated as using the same join path. Plus the original doesn't provide any additional information than before `path` was added to all properties.

```suggestion
        'path' => $base_table . '.' . $base_pkey_col . '>' . $linker_table . '.' . $linker_left_col . ';' . $linker_pkey_col,
```

Reply:
Yes, my original thought is that we should specify the entire join too but in the end it doesn't matter.  We can't guarantee that someone will use the full path to the base table and since we didn't specify that before and I wanted to make as little changes as possible to the fields so ChadoStorage supports it.  I think if we leave it and we ever break something having a mix will ensure we always support it either way.

Reply:
There is also an advantage in using the shorter path length by not including the base table... as it's more readable.

Reply:
But I'll leave it up to you. I really don't care... I'm just nervous about breaking things and having to spend more time on this PR

Reply:
True, I just always think of the fields in core as the "Best Practices" example so want to make sure we give consistent and clear examples 🤔 I lean towards using the full path here but also don't want to break things again 😬 How about we leave these full paths out of this PR. I can make a separate PR that makes them consistent and hopefully that can get in before release but at least by putting them in a separate PR, if something goes wrong it doesn't hold this up.